src/samplers/gamma.jl

```diff
@@ -189,16 +182,16 @@ function GammaMTSampler(g::Gamma)
     GammaMTSampler(d, c, κ)
 end
 
-function rand(s::GammaMTSampler)
+function _rand!(rng::AbstractRNG, s::GammaMTSampler)
     while true
-        x = randn()
+        x = randn(rng)
         v = 1.0 + s.c * x
         while v <= 0.0
-            x = randn()
+            x = randn(rng)
             v = 1.0 + s.c * x
         end
         v *= (v * v)
-        u = rand()
+        u = rand(rng)
         x2 = x * x
         if u < 1.0 - 0.331 * abs2(x2) || log(u) < 0.5 * x2 + s.d * (1.0 - v + log(v))
```

Comment:
should we add a comment on the hard-coded numerical values?

Reply:
Filed this issue separately as  #835 so we can proceed here in the meantime.

Reply:
Thanks! I have no idea what these numbers are, so hard to comment as the code comes from the other PR...